app/src/main/res/layout/activity_event.xml

```diff
@@ -50,7 +50,7 @@
                 android:textCursorDrawable="@null"
                 android:textSize="@dimen/day_text_size" />
 
-            <org.fossify.commons.views.MyEditText
+            <AutoCompleteTextView
```

Comment:
We should use the Fossify implementation so proper colors are used in all themes.

```suggestion
            <org.fossify.commons.views.MyAutoCompleteTextView
```